app/views/components/_image_attachment_table.html.erb

```diff
@@ -3,34 +3,43 @@
   <div class="table-responsive overflow-y-hidden">
     <table class="table mb-0 text-nowrap table-hover table-centered">
       <thead>
-      <tr>
-        <th scope="col"></th>
-        <th scope="col"></th>
-      </tr>
+        <tr>
+          <th scope="col">Image</th>
+          <th scope="col">Filename</th>
+          <th scope="col"></th>
+        </tr>
       </thead>
       <tbody>
-      <% if @pet.images.attached? %>
-        <% @pet.images.each do |image| %>
-          <tr>
-            <td>
-              <div class="d-flex align-items-center">
-                <div class="icon-shape icon-xxxl rounded-3 d-flex flex-column">
-                  <%= image_tag image, class: 'card-img' %>
-                  <%= image.filename %>
+        <% if @pet.images.attached? %>
+          <% @pet.images.each do |image| %>
+            <tr>
+              <td>
+                <div class="d-flex align-items-center">
+                  <div class="icon-shape icon-lg rounded-3">
```

Comment:
Can we use the same class as the demo site? `img-4by3-lg` this makes the image a little larger

Reply:
Sure! Let me update it here.